finance/README.md

```diff
@@ -0,0 +1,157 @@
+# Cloud Spanner Banking Application
```

Comment:
```suggestion
# Cloud Spanner Finance Application
```

Not your change, I know, but this is a good time to update it.

Reply:
Added in #82